python/fledge/services/core/api/service.py

```diff
@@ -682,16 +680,11 @@ async def _put_schedule(protocol: str, host: str, port: int, sch_id: uuid, is_en
             _logger.debug("PUT Schedule response: %s", response)
 
 
-def do_update(http_enabled: bool, host: str, port: int, storage: connect, pkg_name: str, uid: str,
-              schedules: list) -> None:
+def do_update(host: str, port: int, storage: connect, pkg_name: str, uid: str, schedules: list) -> None:
```

Comment:
keep http_enabled as protocol/scheme in signature

Are these duplicate def, across API modules?

Reply:
Plugins and Service Update are dealing differently that's why no common def. Only to avoid over complicated things by passing flag.